src/tasks/reducer.js

```diff
@@ -24,7 +25,8 @@ export function tasksReducer(state = new TasksState(), {payload, type}) {
       return state.merge({
         deleted: null,
         created: payload,
-        list: state.list.unshift(payload)
+        list: state.list.unshift(payload),
+        labelsPool: state.labelsPool.add(Object.keys(payload.label)[0]),
```

Comment:
`state.labelsPool.add(Object.keys(payload.label)[0])` what this line does ?

Reply:
It's adding the labels that are attached to the given task

Reply:
Oops this was a one year bug. I've fixed it now :)

Reply:
Great catch